.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Testspace Action
+on: [push, pull_request]
+
+jobs:
+  units:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm ci
+      - run: npm run lint
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          fetch-depth: 50
+      - name: Testspace client install & config
+        uses: ./
+        with:
+          subdomain: "s2"
+      - name: Testspace push test content
+        run: testspace [${{ matrix.os}}]./testcontent/*.xml
```

Comment:
So if I run the action and there are xml files in the `testcontent` folder they will be uploaded to Testspace?

Reply:
This is just for the CI test, not customer-facing.  Used to push content (pre-canned) for each OS.